spec/features/commenting_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.feature "Comments", type: :feature do
+  let(:dan) { FactoryBot.create :user, name: "dan" }
+
+
+  it "User can comment on post" do
```

Comment:
What about User can't submit empty comment spec? Right now we can see that there is an empty comment being added to the DOM. You should be able to test this by checking the DOM does not load this comment component.

[OPTIONAL]